### DISCUSSION TITLE
Captain announcements should be affected by accent

### DISCUSSION THREAD

**Discussion by Fogapod** [unitystation/unitystation#7894] [Ideas]
### Description
Captain probably makes announcements using their voice so this makes sence.

**Comment by Cboyjet4** [comment]
If its there voice, can mimes not make annoucements then?

**Comment by Fogapod** [comment]
That's a good point actually. If this gets implemented, mimes should not be able to make announcements as well.

**Comment by corp-0** [comment]
mimes can talk, they just choose not to

**Comment by Humonitarian** [comment]
Actually, mimes can't talk unless they break their vow. Following this assumption, mimes making an announcement should break their vow.
